content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,329 @@
-Note: StatsD counts show a decimal value within Datadog, since they are normalized over the flush interval to report units per second.
+The HISTOGRAM metric submission type represents the statistical distribution of a set of values calculated Agent-side in one time interval. Datadog’s HISTOGRAM metric type is an extension of the StatsD timing metric type: the Agent aggregates the values that are sent in a defined time interval and produces different metrics which represent the set of values.
 
-Discover how to submit count metrics:
+If you send `X` values for a HISTOGRAM metric `<METRIC_NAME>` in a given time interval, the following metrics are produced by the Agent by default:
 
-* [With a custom Agent Check][1]
-* [With DogStatsD][2]
-* [With the Datadog API][3]
+| Aggregation                  | Description                                                                                                                                                  | Datadog In-app Type |
+| ---------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------- |
+| `<METRIC_NAME>.avg`          | Represents the average of those `X` values in the time interval.                                                                                             | GAUGE               |
+| `<METRIC_NAME>.count`        | Represents the number of values sampled during the interval, `X`. The Agent submits this number as a RATE so it would show in app the value of `X/interval`. | RATE                |
+| `<METRIC_NAME>.median`       | Represents the median of those `X` values in the time interval.                                                                                              | GAUGE               |
+| `<METRIC_NAME>.95percentile` | Represents the 95th percentile of those `X` values in the time interval.                                                                                     | GAUGE               |
+| `<METRIC_NAME>.max`          | Represents the maximum value of those `X` values sent during the time interval                                                                               | GAUGE               |
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=count
-[2]: /developers/metrics/dogstatsd_metrics_submission/#count
-[3]: /api/?lang=python#post-timeseries-points
-{{% /tab %}}
-{{% tab "RATE" %}}
+**Note**:
 
-**The `RATE` metric submission type represents the number of events over a defined time interval (flush interval) that's normalized per-second.** A `RATE` can be used to track the rate of requests hitting your webservers.
+- Configure which aggregation you want to send to Datadog with the `histogram_aggregates` parameter in your [`datadog.yaml` configuration file][1]. By default, only `max`, `median`, `avg`, and `count` aggregations are sent out to Datadog. `sum` and `min` are also available.
```

Comment:
- Configure which aggregations you want to send to Datadog with the `histogram_aggregates` parameter in your [`datadog.yaml` configuration file][1]. By default, only `max`, `median`, `avg`, and `count` aggregations are sent to Datadog. `sum` and `min` are also available.